tests/console/shells.pm

```diff
@@ -29,6 +34,7 @@ sub run() {
     # ksh does not build for Leap 15.x on aarch64, so, skip it
     push @packages, qw(ksh) unless (is_leap('15.0+') and check_var('ARCH', 'aarch64'));
     zypper_call("in @packages");
+    tcsh_extra_tests();
```

Comment:
I don't think it makes sense to run the "extra tests" before the "basic hello world" test. Can you put that afterwards please?

Reply:
Ah sure, I know why it that and and it sliped:
I was playing around with it because it was failing on opensuse. In the end I discovered that this class was failing even before I touched it. 
Will lower it!